### ISSUE TITLE
FAILED akka.http.impl.engine.ws.MessageSpec

### ISSUE THREAD

**Issue by johanandren** [akka/akka#19995]
https://jenkins.akka.io:8498/job/akka-multi-node/1386/consoleFull

```
- highest bit of 64-bit length is set *** FAILED *** (3 seconds, 64 milliseconds)
[info]     java.lang.AssertionError: timeout (3 seconds) waiting for 1 messages on ErrorFilter(class akka.http.impl.engine.ws.ProtocolException,None,Left(),false)
[info]     at akka.testkit.EventFilter.intercept(TestEventListener.scala:113)
[info]     at akka.http.impl.engine.ws.MessageSpec$$anonfun$1$$anonfun$apply$mcV$sp$58$$anonfun$apply$mcV$sp$63$$anon$48.<init>(MessageSpec.scala:745)
[info]     at akka.http.impl.engine.ws.MessageSpec$$anonfun$1$$anonfun$apply$mcV$sp$58$$anonfun$apply$mcV$sp$63.apply$mcV$sp(MessageSpec.scala:722)
[info]     at akka.http.impl.engine.ws.MessageSpec$$anonfun$1$$anonfun$apply$mcV$sp$58$$anonfun$apply$mcV$sp$63.apply(MessageSpec.scala:722)
[info]     at akka.http.impl.engine.ws.MessageSpec$$anonfun$1$$anonfun$apply$mcV$sp$58$$anonfun$apply$mcV$sp$63.apply(MessageSpec.scala:722)
...
```

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node/1387/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node/1388/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node/1389/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node/1390/consoleFull
https://jenkins.akka.io:8498/job/akka-nightly/3128/consoleFull
https://jenkins.akka.io:8498/job/akka-nightly/3130/consoleFull
https://jenkins.akka.io:8498/job/akka-multi-node-nightly2/288/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly/3131/consoleFull
https://jenkins.akka.io:8498/job/akka-nightly/3132/consoleFull
https://jenkins.akka.io:8498/job/akka-nightly/3133/consoleFull
https://jenkins.akka.io:8498/job/akka-multi-node-nightly2/289/consoleFull

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node/1392/consoleFull

**Comment by johanandren** [comment]
Seems like this started failing with the VirtualProcessor fixes for #19790 

@rkuhn any clues?

**Comment by johanandren** [comment]
More specifically exactly the failing line was changed in that PR: https://github.com/akka/akka/commit/9db36727c0f01b0f074f8ee4557289ac82a2d44b#diff-db141c7376fa2547d3fce9344f2c0916R745

**Comment by rkuhn** [comment]
ah snap, the `push` must also happen within the error filter …

**Comment by rkuhn** [comment]
will submit a fix ASAP

**Comment by rkuhn** [comment]
fix in #20012
